cloudinstall/charms/__init__.py

```diff
@@ -104,6 +104,9 @@ class CharmBase:
     menuable = False
     subordinate = False
     openstack_release_min = 'i'
+    # Set this if a plugin charm overrides a 'system' installed
+    # charm
+    tainted = False
```

Comment:
right, but we're not actually printing the value anywhere now, right? and just having a log message dump the  `__file__` for each of the charms would tell us the same thing